meta-refkit-gateway/recipes-gateway/johnny-five/johnny-five_%.bbappend

```diff
@@ -0,0 +1 @@
+RDEPENDS_johnny-five-serialport-node-pre-gyp-request-node-uuid_remove_df-refkit-gateway = "bash"
```

Comment:
@nagineni is RDEPENDS to bash is also necessary for johnny-five and nodejs? If no required could you remove it on bb. It look more consistent in order to me to create a bbappend on bash remove only.